smart-contracts/contracts/mixins/MixinRoles.sol

```diff
@@ -78,5 +84,33 @@ contract MixinRoles is AccessControlUpgradeable {
     emit KeyGranterRemoved(_granter);
   }
 
+
+  /** `owner()` is provided as an helper to mimick the `Ownable` contract ABI.
+    * The `Ownable` logic is used by many 3rd party services to determine
+    * contract ownership - e.g. who is allowed to edit metadata on Opensea.
+    * 
+    * @notice This logic is NOT used internally by the Unlock Protocol and is made 
+    * available only as a convenience helper.
+   */
+  function owner() public view returns (address) {
+    return _convenienceOwner;
+  }
+
+  /** Setter for the `owner` convenience helper (see `owner()` docstring for more).
+    * @notice This logic is NOT used internally by the Unlock Protocol ans is made 
+    * available only as a convenience helper.
+    * @param account address returned by the `owner()` helper
+   */ 
+  function setOwner(address account) public onlyLockManager {
+    require(account != address(0), 'OWNER_CANT_BE_ADDRESS_ZERO');
+    address _previousOwner = _convenienceOwner;
+    _convenienceOwner = account;
+    emit OwnershipTransferred(_previousOwner, account);
+  }
+
+  function isOwner(address account) public view returns (bool) {
+    return _convenienceOwner == account;
+  }
+
   uint256[1000] private __safe_upgrade_gap;
```

Comment:
So I expect the Gap here to be reduced?

Reply:
Yes, I thought we will do that after other changes where done. But for sure we need a streamlined way to assess the storage continuity

Reply:
Let's actually do that on every PR because I worry we may forget eventually?